system theme causes `undefined is not an object` error

### Description

The system theme is missing from the /theme list. And when I search for the `system` theme, this error shows up:

```
[13:07:56] [ERROR] Error: undefined is not an object (evaluating 'opt.value')
    TypeError: undefined is not an object (evaluating 'opt.value')
        at onMove (src/cli/cmd/tui/component/dialog-theme-list.tsx:26:17)
        at moveTo (src/cli/cmd/tui/ui/dialog-select.tsx:96:11)
        at <anonymous> (src/cli/cmd/tui/ui/dialog-select.tsx:78:9)
```

### Plugins

"plugin": ["opencode-antigravity-auth@beta", "@tarquinen/opencode-dcp@latest"]

### OpenCode version

1.1.35

### Steps to reproduce

1. `opencode`
2. Notices my theme is not system. Open `/theme` and don't see `system` on that list
3. Searches for `system` by typing.
4. Errors shows up

### Screenshot and/or share link

_No response_

### Operating System

macOs 15.7.3 + npm install -g

### Terminal

iTerm2

--- a/packages/opencode/src/cli/cmd/tui/ui/dialog-select.tsx
+++ b/packages/opencode/src/cli/cmd/tui/ui/dialog-select.tsx
@@ -149,7 +149,8 @@ export function DialogSelect<T>(props: DialogSelectProps<T>) {
 
   function moveTo(next: number, center = false) {
     setStore("selected", next)
-    props.onMove?.(selected()!)
+    const option = selected()
+    if (option) props.onMove?.(option)
     if (!scroll) return
     const target = scroll.getChildren().find((child) => {
       return child.id === JSON.stringify(selected()?.value)

